ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -170,6 +170,22 @@ objects:
         cp /tmp/cluster/ssh-privatekey ~/.ssh/kube_openstack_rsa || true
         export TEST_PROVIDER='{"type":"openstack"}'
 
+        # set up openstack env vars for testing
+        function get_clouds_param() {
+          cat /tmp/cluster/clouds.yaml | grep $1: | sed -e "s|.*$1: ||" | tr -d '"'
```

Comment:
Wouldn't this delete double quotes from the values? This could be a problem if we had quotes in our password for example. Here is a more robust example where I replaced cat+grep+sed by awk, and tr by sed:

`awk '/^\s+'$1':/ { print $2 }' /tmp/cluster/clouds.yaml | sed -e 's/^"//' -e 's/"$//'`

Reply:
I added this to the latest version. Thanks.